src/resources/universalJavaApplicationStub.sh

```diff
@@ -0,0 +1,397 @@
+#!/bin/sh
```

Comment:
https://github.com/maven-download-plugin/maven-download-plugin
We can get the most updated version from the internet with Maven as opposed to maintaining the file, licensing, ect..
```
<plugin>
    <groupId>com.googlecode.maven-download-plugin</groupId>
    <artifactId>download-maven-plugin</artifactId>
    <version>1.3.0</version>
    <executions>
        <execution>
			<phase>process-resources</phase>
			<goals>
				<goal>wget</goal>
			</goals>
            <configuration>
                <url>https://raw.githubusercontent.com/tofi86/universalJavaApplicationStub/master/src/universalJavaApplicationStub</url>
                <outputDirectory>resources/</outputDirectory>
            </configuration>
        </execution>
    </executions>
</plugin>
```